src/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -256,6 +257,42 @@ public void onEnable()
 		this.customLogger = new CustomLogger();
         
 		AddLogEntry("Finished loading configuration.");
+
+                // Check we can run with this Minecraft version
+                final String[] supportedVersions = {
+                   "1.11.2", "1.12", "1.12.1"
```

Comment:
Aside from the obvious conflict due to the maven refactor PR I just pulled, this needs to support future versions, or at least only give an info message (i.e. using newer MC versions causes this to not scream as much as using older ones).

Reply:
Especially since it currently is running on the spigot test server, which is running private 1.13 spigot snapshot builds